src/assets/text/scrollyText.js

```diff
@@ -6,71 +6,67 @@ export default {
         },
         {
           id: 'c',
-          text: 'text c: To begin, start with a hydrograph, which is a plot of streamflow over time. Here, we are looking at daily streamflow during the growing season of 1963. '
+          text: 'To begin, we need to consider historical conditions. Over the past 70 years at this site in Ohio, average daily streamflow ranged from 100 to 2700 cfs, with August through October typically having the lowest streamflow during the year.'
         },
         {
           id: 'd',
-          text: 'text d: A \"fixed\" threshold is created by first calculating average streamflow across all years and days and then calculating 10% of this annual average streamflow as the threshold.'
+          text: 'By considering records across the whole year, streamflow levels that fall below a threshold of 45 cfs are in the lowest 10%. This fixed threshold of 45 cfs is constant from day-to-day and identifies drought events only during the dry season when the lowest streamflow levels occur.'
         },
         {
           id: 'e',
-          text: 'text e: That\'s how these periods of drought were defined. These dry-season droughts are especially important to detect for hydropower production and for aquatic ecosystems that rely on water year-round.'
+          text: 'Using a 10% fixed threshold identifies two drought events in 1963 in the dry season. Unusually low flow at this time of year can reduce hydropower production, impair water quality, and require reservoir releases to provide water to communities.'
         },
         {
           id: 'f',
-          text: 'text f: But droughts can also occur in the spring and winter when we expect wetter conditions and greater streamflow from rain showers and snow melt! To understand this, let\'s zoom out and look at the average daily streamflow for the last 70 years or so.'
+          text: 'But drought can also occur during wetter seasons if flow levels are unusually low for that time of year. To identify seasonal drought events, let\'s zoom out and look at the average daily streamflow for the last 70 years.'
         },
         {
           id: 'g',
-          text: 'text g: In 1963, spring streamflow was much lower than a typical spring.'
+          text: 'In 1963, spring streamflow was much lower than in a typical spring but still not as low as during the dry season.'
         },
         {
           id: 'h',
-          text: 'text h: To detect droughts in wetter seasons like this, we have to calculate a variable threshold as 10% of the average daily streamflow.'
+          text: 'To identify drought events during wetter times of the year, a variable threshold can be used to compare streamflow to typical conditions at that time of the year. Streamflow drought occurs if streamflow falls below the lowest 10% of observations ever recorded for that day. The result is a threshold that varies day to day.'
```

Comment:
SC: consider adding either 'With this approach', or 'Here,', or 'With this method,' to the start of the second sentence. I think it might be taken as a blanket statement as is.